xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -514,6 +514,11 @@ bool CGUIWindowVideoBase::ShowInfoAndRefresh(const CFileItemPtr& item, const Scr
     m_viewControl.SetSelectedItem(itemNumber);
   }
 
+  CVariant data;
+  data["id"] = item->GetVideoInfoTag()->m_iDbId;
+  data["type"] = item->GetVideoInfoTag()->m_type;
+  CServiceBroker::GetAnnouncementManager()->Announce(ANNOUNCEMENT::VideoLibrary, "OnUpdate", data);
```

Comment:
I don't doubt this fixes the issue but the announce for videolibrary update is being done by a GUI window when it should be done by the actual library component...
Any change you can find out a proper place for the announce?
Thanks for tackling some of our open issues! It's greatly appreciated

Reply:
I do not know how to fix this properly, but the taken approach seems not the best, like @enen92 pointed out here.

@CrystalP If I remember correctly you already dealt with similar problems. Do you have an idea for a more suitable fix. Using window's `Refresh()` method or something like this maybe?

Reply:
I think that was one of my first PRs #24176 which @CrystalP guided me through.

Reply:
> Refresh() doesn't always get called. 

I meant maybe you could call Refresh() yourself to fix the issue.

Reply:
I could, but that's still in the GUI class - @enen92 preferred it out of there - are you happy with the current solution in VideoDatabase??